README.md

```diff
@@ -130,7 +130,7 @@ openModal({
 Close all open modals.
 
 ```js
-openModal();
+closeModal();
 ```
 
 #### isModalOpen
```

Comment:
Question:

Related to `encodeUrlParams`, is it possible to use modal URL in a link `href` directly, without `router.push`? Eg.

```
<a href=`/account/${encodeUrlParams({ id: user.id })}`>Edit account</a>
```

If so, then it should be documented here too.

Reply:
Yeah, we do that in our platform it would be:

```
<a href=`/account?modal=edit-account&params=${encodeUrlParams({ id: user.id })}`>Edit account</a>
```